docs/assets/diagrams/transfer/03-confirm-payee.puml

```diff
@@ -0,0 +1,118 @@
+@startuml
+
+title Confirm Payee
+
+hide footbox
+
+autonumber
+
+box "Mobile Device" #Light
+  actor Alice
+  participant "PISP App" as App
+end box
+box "PISP" #Light
+  database Firebase
+  participant "PISP Server" as Server
+end box
+box "Mojaloop" #Light
+  participant "Switch" as Mojaloop
+end box
+
+Alice -> App: Initiate this transaction
+
+rnote right of App #Light
+  ""firebase.firestore()""
+  ""  .collection("transactions")""
+  ""  .doc("456")""
+  ""  .set({""
+  ""    sourceAccountId: "bob.fspb",""
+  ""    consentId: 123,""
+  ""    amount: {""
+  ""      amount: 20.00""
+  ""      currency: "USD"""
+  ""    },""
+  ""    payer: {""
+  ""      personalInfo: {""
+  ""        complexName: {""
+  ""          firstName: "Bob",""
+  ""          lastName: "Beaver"""
+  ""        }""
+  ""      },""
+  ""      partyIdInfo: {""
+  ""        partyIdType: "OPAQUE",""
```

Comment:
I'm not sure this should be `OPAQUE` here. We were planning to use `OPAQUE` primarily for the account linking step when we know the dfsp, but _not_ the identifier type.

By this stage, I think we should have a proper identifier and type (e.g. `EMAIL`, `MSISDN` etc.) but the question remains: when do we actually get the "proper" mojaloop identifier?

Reply:
Do we get the proper identifier and type from account linking?

Reply:
Updated to use MSISDN for now

Reply:
Good question. I'm actually not sure. Initially we assumed we would have a _proper_ lookup in the beginning of the account linking, and so would have already saved the `partyIdType` and `partyIdentifier`. That has now changed (as you can tell).

Perhaps we can rely on the result (the `PUT /parties/OPAQUE/{id}`) from the `OPAQUE` account lookup at the start of account linking to have a concrete `partyIdType` (e.g. https://docs.mojaloop.io/mojaloop-specification/documents/API%20Definition%20v1.0.html#7411-party) in the request body.

@jgeewax  thoughts?

Reply:
okay, will create a ticket for it and update this comment